server/bin/pbench-tarball-stats.py

```diff
@@ -158,27 +188,29 @@ def transform_buckets(buckets: dict, sat_names: list):
     Bad Tar Balls:
         {{ "{:18n}".format(bad.ctrls.keys()|length) }} controllers
         {{ "{:18n}".format(bad.count) }} count
-        {{ "{:18n}".format(bad.size) }} size (bytes)
+        {{ "{:>18s}".format(bad.size|humanize_naturalsize) }} size
 
     {% endif %}
 {% if server_origin.keys()|length > 0 %}
-{% if by_week.keys()|length > 0 %}
-Tar Ball Counts broken down by weeks, with satellite percentages:
+Tar Ball Counts broken down by weeks for most recent 2 months (with satellite percentages):
 
 Week Ending  Total Count{% for name in server_origin.keys()|sort %}   {{ "% {0:>5s}".format(name) }}{% endfor +%}
-{% for name,value in by_week.items()|sort %}
-{{ "{0:<10s}".format(name) }}   {{ "{:11n}".format(value.count) }}{% for name in server_origin.keys()|sort %}   {{ "{:6.2f}%".format(value.sats_pct[name]) }}{% endfor +%}
+{% for name,value in by_week.items()|sort(reverse=True) %}
+{{ "{0:<10s}".format(name) }} {% if value.partial %}P{% else %} {% endif +%} {{ "{:11n}".format(value.count) }}{% for name in server_origin.keys()|sort %}   {{ "{:6.2f}%".format(value.sats_pct[name]) }}{% endfor +%}
 {% endfor %}
-{% endif %}
 
-Tar Ball Counts broken down by Year and Month, with satellite percentages:
+Tar Ball Counts broken down by Year (with satellite percentages):
 
-Year/Month   Total Count{% for name in server_origin.keys()|sort %}   {{ "% {0:>5s}".format(name) }}{% endfor +%}
-{% for name,value in by_year.items()|sort %}
-{{ "{0:<10s}".format(name) }}   {{ "{:11n}".format(value.count) }}{% for name in server_origin.keys()|sort %}   {{ "{:6.2f}%".format(value.sats_pct[name]) }}{% endfor +%}
+Year         Total Count{% for name in server_origin.keys()|sort %}   {{ "% {0:>5s}".format(name) }}{% endfor +%}
```

Comment:
Since you're including the percent sign in the values, I don't think you need it in the header (and it looks funny together with the subsequent right-justified header value).

Ditto lines 197 and 211.